src/Microsoft.ML.Tokenizers/Microsoft.ML.Tokenizers.csproj

```diff
@@ -21,4 +21,38 @@
     <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
   </ItemGroup>
 
+  <UsingTask TaskName="CompressFile"
+    TaskFactory="RoslynCodeTaskFactory"
+    AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll" >
+    <ParameterGroup>
+      <Files ParameterType="Microsoft.Build.Framework.ITaskItem[]" Required="true" />
+    </ParameterGroup>
+    <Task>
+      <Using Namespace="System.IO" />
+      <Using Namespace="System.IO.Compression" />
+      <Code Type="Fragment" Language="cs">
+        foreach(var file in Files)
+        {
+            using var sourceStream = File.OpenRead(file.GetMetadata("FullPath"));
+            using var destStream = new DeflateStream(File.Create(file.GetMetadata("Destination")), CompressionLevel.Optimal);
+            sourceStream.CopyTo(destStream);
+        }
+      </Code>
+    </Task>
+  </UsingTask>
+
+  <ItemGroup>
+    <FilesToCompress Include="Data\**\*.*" Destination="$(IntermediateOutputPath)%(FileName).zip"/>
```

Comment:
Don't use zip. These are not zips and naming them so is confusing.  Consider `.deflate`

Reply:
I fixed that.